engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -40,6 +40,7 @@ namespace dmModelImporter
 struct GltfData
 {
     cgltf_data* m_Data;
+    uint8_t     m_Finalized : 1;
```

Comment:
The load flow is multi step, and spread out, so I opted for this check.

Reply:
I don't quite understand the fix. Is `LoadFinalizeGltf` called multiple times? If so, why is that?